assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -104,6 +104,19 @@ const Inner = styled.div`
     / 1fr;
 `;
 
+const Loading = styled.div`
+  position: absolute;
+  bottom: 20px;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: 8px 80px;
+  background-color: rgba(0, 0, 0, 0.4);
```

Comment:
Let's use the theme color here, and `rgba`.

Something like this:

```suggestion
  background-color: ${({ theme }) => rgba(theme.colors.bg.v0, 0.4)};
```

requires `import { rgba } from 'polished';` though.